build-system/SERVING.md

```diff
@@ -0,0 +1,59 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# How AMP HTML is deployed
+
+## Requirements
+- git
+- node + npm
+- gulp (installed globally)
+
+## Steps
+```bash
+git clone https://github.com/ampproject/amphtml.git
+cd amphtml
+# Checkout a tag
+git checkout 123456789
+npm install
+gulp clean
+# We only need to build the css files, no need to generate `max` files
+gulp build --css-only
+gulp dist --version 123456789 --type prod
+mkdir -p /path/to/cdn/production/
+mkdir -p /path/to/cdn/3p/
+# this would be the files hosted on www.ampproject.org/
+cp -R dist/* /path/to/cdn/production/
+# this would be the files hosted on 3p.ampproject.net/
+cp -R dist.3p/* /path/to/cdn/3p
+
+# Unfortunately we need these replace lines to compensate for the transition
+# to the new code. We should be able to remove this in the next couple of weeks
+# as we no longer prefix the global AMP_CONFIG during `gulp dist` in the latest
+# code in master. We use -i.bak for cross compatibility on GNU and BSD/Mac.
+sed -i.bak "s#^.*\/\*AMP_CONFIG\*\/##" /path/to/cdn/production/v0.js
```

Comment:
@dknecht unfortunately had to add this as we're transitioning the code base to the new code. this makes sure we don't duplicate the config at the beginning of the file.

the new code i've added here will sanity check it but since we checkout an old tag to replicate a build we might checkout code that doesnt have this new code yet.